packages/@aws-cdk/aws-glue-alpha/lib/table.ts

```diff
@@ -328,6 +381,11 @@ export class Table extends Resource implements ITable {
           serdeInfo: {
             serializationLibrary: props.dataFormat.serializationLibrary.className,
           },
+          parameters: props.storageParameters ? props.storageParameters.reduce((acc, param) => {
+            const key = param.key.key;
+            acc[key] = param.value.value;
+            return acc;
+          }, {} as { [key: string]: string }) : undefined,
```

Comment:
We need to validate and throw if the same parameter is used multiple times. This makes `storageParameters ` kind of like a map.

Reply:
Yep, makes sense. I'll work on it